### ISSUE TITLE
ImportError: No module named SimpleAdminConfig

### ISSUE THREAD

**Issue by bashu** [idlesign/django-siteprefs#13]
in settings:
```python
INSTALLED_APPS = [
    ...
    'siteprefs',
    'adminplus',  
    'django.contrib.admin.apps.SimpleAdminConfig',
    ...
]
```
I got this
```python
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/wsgiref/handlers.py", line 85, in run
    self.result = application(self.environ, self.start_response)
  File "/Users/bashu/Projects/django-sigma-cms-demo/virtualenv/lib/python2.7/site-packages/django/contrib/staticfiles/handlers.py", line 63, in __call__
    return self.application(environ, start_response)
  File "/Users/bashu/Projects/django-sigma-cms-demo/virtualenv/lib/python2.7/site-packages/django/core/handlers/wsgi.py", line 170, in __call__
    self.load_middleware()
  File "/Users/bashu/Projects/django-sigma-cms-demo/virtualenv/lib/python2.7/site-packages/django/core/handlers/base.py", line 52, in load_middleware
    mw_instance = mw_class()
  File "/Users/bashu/Projects/django-sigma-cms-demo/virtualenv/lib/python2.7/site-packages/django/middleware/locale.py", line 24, in __init__
    for url_pattern in get_resolver(None).url_patterns:
  File "/Users/bashu/Projects/django-sigma-cms-demo/virtualenv/lib/python2.7/site-packages/django/core/urlresolvers.py", line 402, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/Users/bashu/Projects/django-sigma-cms-demo/virtualenv/lib/python2.7/site-packages/django/core/urlresolvers.py", line 396, in urlconf_module
    self._urlconf_module = import_module(self.urlconf_name)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/Users/bashu/Projects/django-sigma-cms-demo/djangoapp/demo/urls/development.py", line 1, in <module>
    from .base import *
  File "/Users/bashu/Projects/django-sigma-cms-demo/djangoapp/demo/urls/base.py", line 8, in <module>
    autodiscover_siteprefs()
  File "/Users/bashu/Projects/django-sigma-cms-demo/virtualenv/lib/python2.7/site-packages/siteprefs/toolbox.py", line 97, in autodiscover_siteprefs
    import_prefs()
  File "/Users/bashu/Projects/django-sigma-cms-demo/virtualenv/lib/python2.7/site-packages/siteprefs/utils.py", line 231, in import_prefs
    import_module(app, PREFS_MODULE_NAME)
  File "/Users/bashu/Projects/django-sigma-cms-demo/virtualenv/lib/python2.7/site-packages/siteprefs/utils.py", line 208, in import_module
    module = import_module_(package)
  File "/Users/bashu/Projects/django-sigma-cms-demo/virtualenv/lib/python2.7/site-packages/django/utils/importlib.py", line 45, in import_module
    __import__(name)
ImportError: No module named SimpleAdminConfig
```

**Comment by idlesign** [comment]
Thank you for the report.

That one is related to app configs, so we should handle those cases separately: e.g. for the case above we'd rsther not try to import `'django.contrib.admin.apps.SimpleAdminConfig'` as it is but need to try to discover `settings.py` under `django.contrib.admin`.

**Comment by idlesign** [comment]
Seems to be fixed in 0.5.3 (through `django-etc` update).

**Comment by bashu** [comment]
nice!
